addOns/custompayloads/src/main/java/org/zaproxy/zap/extension/custompayloads/CustomMultiplePayloadDialog.java

```diff
@@ -53,7 +56,35 @@ public CustomMultiplePayloadDialog(Window owner, CustomPayload payload) {
     public void addFileButtonListener(JButton fileButton) {
         fileButton.addActionListener(
                 e -> {
-                    JFileChooser chooser = new JFileChooser();
```

Comment:
Why duplicate `ReadableFileChooser`? This still does not address the problem of allowing to import nothing.

Reply:
Hah, I hadn't meant to duplicate it, I'll add commonlib and use it.

I didn't realize you were also after an import summary, I'll look at doing that too.

Reply:
No, not wanting that, the problem is that we allow to import literally nothing, open the import and press Add, it closes the dialogue and does nothing. Instead of doing nothing it should inform that a file is required (or disable the Add button and only enable it after the user has selected a file).